tests/UnifiedSpecTests/Operation.php

```diff
@@ -769,6 +769,10 @@ private function executeForSession(Session $session)
     private function executeForBucket(Bucket $bucket)
     {
         $args = $this->prepareArguments();
+
+        // "md5" field is removed from the spec, option "disableMD5" is ignored
+        unset($args['disableMD5']);
```

Comment:
Is this needed to allow running `upload-disableMD5.yml`, which still exists in the upstream repository?

Those tests all assert `md5: { $$exists: false }`, so I'm not sure what it would matter. Wouldn't PHPLIB 2.x just ignore the option (vs. a possible deprecation notice in 1.x)?

Reply:
I likely misunderstood the point of this change. Is it just being done to satisfy the [Util.php](https://github.com/mongodb/mongo-php-library/blob/v1.x/tests/UnifiedSpecTests/Util.php) check for unsupported options? If so, it'd be helpful to make a note of that in this comment (since I've just demonstrated how likely I am to misunderstand the purpose of this `unset()` ^_^).

Reply:
After investigations, `disableMD5` option is [documented in the spec](https://github.com/mongodb/specifications/blob/9058491f2d5b5651913a56e373121a24d6498673/source/gridfs/gridfs-spec.md?plain=1#L281-L288). So I added the option to the list of allowed options, with a comment.

Reply:
So I restored the option `disableMD5` that I removed from the allowed options in https://github.com/mongodb/mongo-php-library/pull/1398

Reply:
If spec tests actually use a `disableMD5` option for upload operations, I think that's an oversight in the spec itself. The option should probably be added to [File Upload](https://github.com/mongodb/specifications/blob/master/source/gridfs/gridfs-spec.md#file-upload) with the same note it has on the corresponding bucket option (that it's deprecated and transitional).

Defer to you if that's worth doing at this point.